diff --git a/dist/client/index.js b/dist/client/index.js
index 2def4d831ec7b70ea467db29b134adecdc2d8643..f999ff87627d5e48f52a25bb0c697a78d225a012 100644
--- a/dist/client/index.js
+++ b/dist/client/index.js
@@ -512,7 +512,11 @@ function doRender(input) {
         if (input.scroll) {
             const { x , y  } = input.scroll;
             (0, _handlesmoothscroll.handleSmoothScroll)(()=>{
-                window.scrollTo(x, y);
+                if ('__scrollTo' in window) {
+                    window.__scrollTo(y);
+                } else {
+                    window.scrollTo(x, y);
+                }
             });
         }
     }