diff --git a/dist/client/index.js b/dist/client/index.js
index adb4ea2f65626abab9f92e5ef6ddd2a1f8d2562c..6cb2068c5def542d0cfa7aa51d001545bee0366e 100644
--- a/dist/client/index.js
+++ b/dist/client/index.js
@@ -476,7 +476,11 @@ function doRender(input) {
             // Otherwise it will not pickup the change in scrollBehavior
             // More info here: https://github.com/vercel/next.js/issues/40719#issuecomment-1336248042
             htmlElement.getClientRects();
-            window.scrollTo(input.scroll.x, input.scroll.y);
+	    if ('__scrollTo' in window) {
+	        window.__scrollTo(input.scroll.y);
+	    } else {
+	        window.scrollTo(input.scroll.x, input.scroll.y);
+	    }
             htmlElement.style.scrollBehavior = existing;
         }
     }